src/OSPSuite.Core/Domain/Builder/ExpressionProfileBuildingBlock.cs

```diff
@@ -76,7 +59,25 @@ public override void UpdatePropertiesFrom(IUpdatable source, ICloneManager clone
          Name = sourceExpressionProfile.Name;
 
          _initialConditions.Clear();
-         sourceExpressionProfile.InitialConditions.Each(initialCondition => AddInitialCondition(cloneManager.Clone(initialCondition)));
+         sourceExpressionProfile.Each<InitialCondition>(initialCondition => Add(cloneManager.Clone(initialCondition)));
```

Comment:
Same here. Each with an <> is very unexpected code. I think I want the old code back